### ISSUE TITLE
Photos on websites arent showing

### ISSUE THREAD

**Issue by Santeri524** [microsoft/vscode#108050]
![ei toimi](https://user-images.githubusercontent.com/72325392/95004589-c2e75900-05f5-11eb-807c-6540fa437e4a.JPG)
Issue Type: <b>Bug</b>

I have tried to get photos to a website and they arent showing only the alt text and there is a somekind of logo before it where the image should be in.

VS Code version: Code 1.49.3 (2af051012b66169dde0c4dfae3f5ef48f787ff69, 2020-10-02T17:57:01.595Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6400 CPU @ 2.70GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.94GB (1.63GB free)|
|Process Argv|--crash-reporter-id 95bed44d-189d-4075-81eb-ab871d684955|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
php-debug|fel|1.13.0
php-intellisense|fel|2.3.14

</details>

**Comment by nrayburn-tech** [comment]
Some more information is needed.
Is it a Microsoft website? A website you’re developing?

It looks like you’re making a website, but aren’t using the correct path. That’s why you’re seeing the icon in front of the alt text. This doesn’t seem like a vscode issue.

**Comment by mjbvz** [comment]
Closing as this sounds like a general development question instead of a VS Code issue
